src/components/breadcrumb/Breadcrumb.tsx

```diff
@@ -0,0 +1,67 @@
+/**
+ * Copyright (c) 2022 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+import { MouseEventHandler, ReactNode } from "react";
+import "styles/components/Breadcrumb.scss";
+import Breadcrumb from 'react-bootstrap/Breadcrumb';
+
+/**
+ * Props for the doughnut chart
+ * @param {title} is to get the title of the Breadcrumb Item.(can be string or JSX Element)
+ * @param {path} is to get the path on which url should be redirected on clicking of particular Breadcrumb Item.
+ * @param {isActive} is to get the status of the Breadcrumb Item whether its active or not.
+ * @param {breadcrumbItems} is to get the array of Breadcrumb Elements to be displayed in the Breadcrumb.
+ * @param {className} is to get a custom class to be applied on Breadcrumb or else default class is applied.
+ * @param {onClickHandler} is to get a custom handler to be applied on click of Breadcrumb Element.
```

Comment:
Please add documentation above the appropriate type. If you want you can refer to code from https://github.com/EMCECS/clarity-react

Reply:
done